### ISSUE TITLE
Cannot configure `timeToWaitForATTUserAuthorization` for `AppsflyerPlugin`

### ISSUE THREAD

**Issue by ebeigarts** [segmentio/analytics-react-native#926]
Currently there is no way to configure `timeToWaitForATTUserAuthorization` for `AppsflyerPlugin`. It is hardcoded as `60`.
In our app the tracking prompt is not triggered on the first screen, so we want to increase the timeout.

https://github.com/segmentio/analytics-react-native/blob/master/packages/plugins/plugin-appsflyer/src/AppsflyerPlugin.tsx#L28

**Expected behavior**

```
new AppsflyerPlugin({ timeToWaitForATTUserAuthorization: 120 })
```

**Comment by oscb** [comment]
I'll add an option for this in the configuration.

**Comment by alanjcharles** [comment]
addressed in #945
